root.sh

```diff
@@ -37,13 +37,12 @@ incremental_recipe: |
   JOBS=$((${JOBS:-1}*3/5))
   [[ $JOBS -gt 0 ]] || JOBS=1
 
-  cmake --build . -- ${JOBS:+-j$JOBS} install
-  mkdir -p $INSTALLROOT/etc/modulefiles && rsync -a --delete etc/modulefiles/ $INSTALLROOT/etc/modulefiles
+  cmake --build . --target install ${JOBS:+-- -j$JOBS}
+  rm -vf "$INSTALLROOT/etc/plugins/TGrid/P010_TAlien.C"         \
```

Comment:
How is this related to gfortran ?

Reply:
It's not, and it's not changing the behaviour (those are syntax improvements), I won't open a second PR just for that 😄